src/brain/issueLabelHandler/followups.ts

```diff
@@ -77,8 +77,7 @@ export async function updateCommunityFollowups({
     return;
   }
 
-  const owner = payload.repository.owner.login;
-  const octokit = await getClient(ClientType.App, owner);
+  const octokit = await getClient(ClientType.App, app.org);
```

Comment:
This is a nice deduplication, we already have to parse out the org in `loadAppFromPayload` so we can centralize that logic (see below) and reuse the result.